Automattic/jetpack
# Carousel: remove unused static images and tweak RTL styles

Maybe resolves https://github.com/Automattic/jetpack/issues/20221 and resolves https://github.com/Automattic/view-design/issues/323

#### Changes proposed in this Pull Request:

Removing images directory, references to images and non-existent elements.

We're also tweaking arrows in RTL, and removing the `rtl/jetpack-carousel-rtl.css`, which seems to be referenced nowhere.

 Things currently look like this in RTL:

<img width="500" alt="Screen Shot 2021-07-08 at 4 35 45 pm" src="https://user-images.githubusercontent.com/6458278/124874015-8e804d80-e00a-11eb-833e-8bf52b85e239.png">

**Questions:**
- It looks like RTL styles are built automatically when we build. Is the stylesheet `projects/plugins/jetpack/modules/carousel/rtl/jetpack-carousel-rtl.css` redundant/have any effect at all?


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions

Check that no icons/images are missing.

In an RTL language, make sure things work and look as they should.

**LTR**
<img width="600" alt="Screen Shot 2021-07-08 at 4 28 21 pm" src="https://user-images.githubusercontent.com/6458278/124873741-347f8800-e00a-11eb-9f09-43861dc4d234.png">

**RTL**

<img width="600" alt="Screen Shot 2021-07-08 at 4 27 28 pm" src="https://user-images.githubusercontent.com/6458278/124873733-334e5b00-e00a-11eb-9824-acca72046dee.png">
